src/SimplygonFunctionApp/AzRemeshFn.cs

```diff
@@ -92,11 +92,13 @@ public static class AzRemeshFn
             log.LogInformation("Cleaning up ... ");
 
             // Cleanup by deleting the directories
-            try {
+            try
+            {
                 outputDir.Delete(true);
                 // Directory.Delete(zipDir, true); // this likely throws an exception (dangling file pointer?) but the exception is not caught by the `catch` block
             }
-            catch(Exception ex) {
+            catch (Exception ex)

```

Comment:
Catch only the exceptions that matters. In this case, 4 of them I guess: https://docs.microsoft.com/fr-fr/dotnet/api/system.io.directory.delete?view=netcore-3.1
```csharp
catch (Expression ex) when (ex is IOException || ex is UnauthorizedAccessException || ex is DirectoryNotFoundException || ex is PathTooLongException)
```